src/Sylius/Bundle/ProductBundle/Doctrine/ORM/ProductAttributeValueRepository.php

```diff
@@ -13,6 +13,8 @@
 
 namespace Sylius\Bundle\ProductBundle\Doctrine\ORM;
 
+use Doctrine\DBAL\Connection;
```

Comment:
Not used anymore

Reply:
Yeah, there are a few obsolete imports, missing strict types declarations, code style issues etc. I just ignored them since the automated ecs fixer will take care of them 💃